src/sagas/trendingSaga.js

```diff
@@ -0,0 +1,19 @@
+import { call, put, takeLatest } from 'redux-saga/effects'
```

Comment:
Do you notice a pattern in `searchSaga` and `trendingSaga` ?? Could that be abstracted so our code is comply with DRY?

Reply:
I did not notice the similarity until now. Yes that code can be moved to an utility method that creates the generator passing the different actions and giphy api details.